solana/bridge/program/src/lib.rs

```diff
@@ -56,6 +58,7 @@ pub use api::{
     InitializeData,
     PostMessage,
     PostMessageData,
+    PostMessageUnreliable,
```

Comment:
naming nit: If I was using a library and I saw a method called `*_unreliable` I probably wouldn't even consider using it, especially since there are no doc comments explaining what exactly is unreliable about it.  How about something like `post_message_reusable`?

Reply:
Added a comment; we want to warn users and only have them use it if they know what they're doing.